frontend/src/metabase/home/components/Onboarding/OnboardingAccordion.module.css

```diff
@@ -0,0 +1,88 @@
+.item:not([data-css-specificity-hack="🪗"]) {
```

Comment:
Extracted `Onboarding.module.css` into this `OnboardingAccordion.module.css` file so we can share it with both onboarding accordions. Please ignore the following CSS file as it's just extracted as-is.